<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Graph</class>
 <widget class="QDialog" name="Graph">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1249</width>
    <height>965</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{background-image: url(:/res/img/fon22.jpg);}
QGroupBox{background-color: rgb(232, 246, 255);}



QPushButton {
color: rgb(255, 255, 255);
     border-radius: 30px;
     background-color: rgb(81, 78, 120);
     min-width: 80px;

 } 
 QPushButton:pressed {
	background-color: rgb(104, 93, 145);
 }

 QPushButton:flat {
     border: none; /* для плоской кнопки границы нет */
 }
QPushButton::hover {
    background-color: rgb(115, 104, 161);
}

QLabel{color: rgb(50, 48, 74);}</string>
  </property>
  <widget class="QOpenGLWidget" name="outputGraph">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>1191</width>
     <height>791</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>820</y>
     <width>651</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>401</width>
      <height>88</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="1">
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <family>Microsoft YaHei UI</family>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <family>Microsoft YaHei UI</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <family>Microsoft YaHei UI</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_1">
       <property name="font">
        <font>
         <family>Microsoft YaHei UI</family>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="result_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Вывести результат</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
